README.md

```diff
@@ -31,7 +31,7 @@ Interviews
    pointing to the next node by means of a pointer. It is a data structure
    consisting of a group of nodes which together represent a sequence.
  * **Singly-linked list**: linked list in which nodes have pointers to the next node and the last node points to null
- * **Doubly-linked list**: linked list in which nodes have two pointers to points to *both* the previous node and the next node
+ * **Doubly-linked list**: linked list in which nodes have two pointers which points to *both* the previous node and the next node
```

Comment:
@EugenHotaj  
 > Change "points" to "point" (only in this sentence, the rest are ok).

Which sentence?